build_package.py

```diff
@@ -567,6 +568,7 @@ def build_common_docker_and_package_files(create_initd_link, base_configs=None):
     make_directory("root/etc/init.d")
     make_directory("root/var/log/scalyr-agent-2")
     make_directory("root/var/lib/scalyr-agent-2")
+    make_directory("root/var/lib/scalyr-agent-2/checkpoints")
```

Comment:
Ok, I missed this during the review.  You are changing the directory where the checkpoints are stored?  Any particular reason?  I would vote against this unless there is a really good reason.

Reply:
Sorry, I forgot to say you that there was no checkpoints folder before this PR. There were just 2 files - `checkpoints.json` and `active-checkpoints.json` directly in the agent `data` folder. 
I introduced the `checkpoints` directory to not flood the data folder with the worker checkpoint files.

Reply:
Sorry, I didn't see this response until just now.  Still, this is a data folder just for the Scalyr Agent.  I think it is fine to put all of the checkpoint files in it.  Introducing another directory just adds a little more complication.  For example, K8s needs to mount the checkpoint directory to a persistent place on the system.  This probably wouldn't break this, but still, I'd rather not take the chance.